front/tests/007-project-delete.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightHomePage } from './home-page-model';
+import project from './assets/operationStudies/project.json';
+
+import { ProjectPage } from './pages/project-page-model';
+
+test.describe('Test is operationnal study: study creation workflow is working properly', () => {
+  test('Delete project', async ({ page }) => {
+    const playwrightHomePage = new PlaywrightHomePage(page);
+    const projectPage = new ProjectPage(page);
+
+    await playwrightHomePage.goToHomePage();
+    await playwrightHomePage.goToOperationalStudiesPage();
+
+    await projectPage.clickProjectByName(project.name);
+    await projectPage.checkLabelProjectSelected('1 projet sélectionné');
```

Comment:
Can you also test that the project can be deleted from its page ? To do in a future PR